core/server/data/migrations/versions/4.0/10-refactor-unsplash-setting.js

```diff
@@ -0,0 +1,29 @@
+const {createIrreversibleMigration} = require('../../utils');
+
+module.exports = createIrreversibleMigration(async (knex) => {
+    const unsplashSetting = await knex('settings')
```

Comment:
Daniel reminded me on another PR, we probs wanna ass a little log message for visibility!

Reply:
Yeeees, agreed :relaxed: